spec/system/qualifications/user_gives_feedback_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+require "rails_helper"
+
+RSpec.feature "Feedback", host: :check_records, type: :system do
```

Comment:
There's Check-specific code in this spec. It'll need going through and reworking so that it's executing against AYTQ.